packages/site/src/hooks/useRequest.ts

```diff
@@ -0,0 +1,43 @@
+import type { RequestArguments } from '@metamask/providers/dist/BaseProvider';
```

Comment:
This is exported from the root in the new version.
```suggestion
import type { RequestArguments } from '@metamask/providers';
```